src/analysis/classic/hunter/CONFIG.tsx

```diff
@@ -1,15 +1,22 @@
+import { bdfreeman1421 } from 'CONTRIBUTORS';
 import Expansion from 'game/Expansion';
 import SPECS from 'game/SPECS';
+import { Icon } from 'interface';
 import Config from 'parser/Config';
 
 import CHANGELOG from './CHANGELOG';
 
+export enum Build {
```

Comment:
we are trying to steer away from using builds - see the examples in the `_templates` folder.

This file and a lot of the other files should be in a folder named `survival`. Shared files should be under `shared`.

Reply:
Done